<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>markingscheme</class>
 <widget class="QDialog" name="markingscheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207)</string>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lecturer in charge</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Assignment Date </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Semester</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_level">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_sem">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_subject">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Computer Programming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Introduction to Computers and Operating Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java Programming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Object Oriented Programming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Database Management Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data Structures and Algorhythms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Architecture and Compiler Design</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rapid Application Development</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data communication and Computer Networks</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Software Engineering</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Web Designing and e-commerce</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mobile and Cloud Computing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Image Processing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Artificial Intelligence</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Parallel Computing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Data Mining</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>2</month>
     <day>21</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;vnvbnvbn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a40000;&quot;&gt;vnb vbnvnn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#204a87;&quot;&gt;Add Marking Scheme&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Asignment No.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_assignmentno">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_lecturer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mr. J.R.K.C Jayakody</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mr. V.G.T.N. Vidanagama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mr. T Arudchelvam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mrs. B.Munasinghe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mrs. H.R.K. Nagahamulla</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mrs. R.P.T.H. Gunasekara</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mr. N.D. Liyanagedara</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mrs. W.H.C. Wickramaarachchi</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select the Marking Scheme</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
